src/shared/components/Header/ui/Messages/Messages.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  ActionIcon,
+  Avatar,
+  Flex,
+  Indicator,
+  Menu,
+  Stack,
+  Text,
+  Tooltip,
+  useMantineColorScheme,
+  useMantineTheme
+} from '@mantine/core';
+import { IconMessageCircle } from '@tabler/icons-react';
+
+import { ICON_SIZE, MESSAGES } from './data';
+
+export const Messages = () => {
+  const { colorScheme } = useMantineColorScheme();
+  const theme = useMantineTheme();
+  const messages = MESSAGES.map((m) => (
+    <Menu.Item
+      key={m.id}
+      style={{
+        borderBottom: `1px solid ${
+          colorScheme === 'dark' ? theme.colors.gray[7] : theme.colors.gray[3]
```

Comment:
try use [style-prop](https://mantine.dev/styles/styles-overview/#style-prop) with css variables and read about using [data-mantine-color-scheme-attribute](https://mantine.dev/theming/color-schemes/#data-mantine-color-scheme-attribute) or use `light-dark`